templates/gitlab.rb.j2

```diff
@@ -19,6 +19,15 @@ nginx['redirect_http_to_https'] = {{ gitlab_redirect_http_to_https }}
 nginx['ssl_certificate'] = "{{ gitlab_ssl_certificate }}"
 nginx['ssl_certificate_key'] = "{{ gitlab_ssl_certificate_key }}"
 
+{% if gitlab_letsencrypt == true %}
```

Comment:
Can drop the `== true`. Also, maybe always print the next line ('enable' line) and put this if condition on the line after?